wrapanapi/systems/lenovo.py

```diff
@@ -112,6 +120,29 @@ def list_servers(self):
         self._servers_list = inventory
         return inventory
 
+    def list_switches(self):
+        switch_list = []
+        instance = self._service_instance(LenovoSwitch.API_PATH)
+        raw_switches = instance.get("switchList", None)
+        if raw_switches:
```

Comment:
Looking good, but I'd like to suggest condensing this just a bit, hopefully without losing readability.

I would default the `.get('switchList', [])` call to an empty list, so that there's no need for the if statement - iteration over an empty list does nothing.

If we remove that, and then also remove assigning the `switch` name and directly pass the `LenovoSwitch()` instantiation into the append call, this can be rewritten as list comprehension.

```
# this first line might break 100 chars though
raw_switches = self._service_instance(LenovoSwitch.API_PATH).get('switchList', [])
return [LenovoSwitch(data, system=self) for data in raw_switches]  # going to be empy list, or list of switch objects
```